### DISCUSSION TITLE
Labels of Advanced analytics stats in the chart legend can't be changed

### DISCUSSION THREAD

**Discussion by petkoaa** [apache/superset#32984] [Ideas]
## Screenshot

![image](https://github.com/user-attachments/assets/b31bcdcf-88da-49d5-b784-086269ba7a50)

## Description

The labels that appear in the chart legend for Advanced analytics cant be changed and are just the Metric name prefixed with "percentage_", etc. 

I'd like to either add a way for the user to set the labels for themselves in the UI, or change them so they look less like "raw".  For example "Percentage of" or just "Percentage"

## Design input
I´d like to solve the issue myself, but would like some codepointers to begin. 
Thanks! 

`design:suggest`

**Comment by rusackas** [comment]
Hi @petkoaa - sorry this went silent for so long. 

We're open to a PR, but I haven't gone spelunking for this code either... You're just gonna have to search around, and/or ask one of the community AI bots (on the website, or here on the thread) I've just moved this to a Discussion since it's not really a bug, hopefully others may chime in. 

Meanwhile, perhaps @dosu-bot has some ideas.
